src/Combinatorics/Matroids/realization_space.jl

```diff
@@ -0,0 +1,989 @@
+
+
+mutable struct MatroidRealizationSpace
+    defining_ideal::Union{Ideal, NumFieldOrdIdl, Nothing}
+    inequations::Union{Vector{Oscar.RingElem},Nothing}
+    ambient_ring::Union{Oscar.MPolyRing, Ring, Nothing}
```

Comment:
Having both `MPolyRing` and `Ring` in there is kinda redundant... Also can't you drop the `Oscar.` here and elsewhere?

Reply:
Fixed.